website/blog/2022-04-19-dbt-cloud-postman-collection.md

```diff
@@ -0,0 +1,184 @@
+
+*Below this you’ll find a series of example requests - use these to guide you or [check out the Postman Collection](https://dbtlabs.postman.co/workspace/Team-Workspace~520c7ac4-3895-4779-8bc3-9a11b5287c1c/request/12491709-23cd2368-aa58-4c9a-8f2d-e8d56abb6b1dlinklink) to try it out yourself.*
+
+## Appendix 
+
+###Examples of how to use the Postman Collection
+
+Let’s run through some examples on how to make good use of this Postman Collection.
+
+#### Migrating dbt Cloud Projects
+
+One common question we hear from customers is “How can we migrate resources from one dbt Cloud project to another?” Often, they’ll create a development project, in which users have access to the UI and can manually make changes, and then migrate selected resources from the development project to a production project once things are ready.
+
+There are several reasons one might want to do this, including: 
+
+- Probably the most common is separating dev/test/prod environments across dbt Cloud projects to enable teams to build manually in a development project, and then automatically migrate those environments & jobs to a production project.
+- Building “starter projects” they can deploy as templates for new teams onboarding to dbt from a learning standpoint.
+- Some particularly security conscious customers may require version control and governance processes to run before any change to infrastructure, including their dbt Cloud accounts.
+
+Below, we show an example of migrating an environment and a job definition from a development project to a production project. Note that this assumes you already have both projects set up. You could take this automation process even further by creating new projects with repository and database connections via the API as well. Please refer to the collection for additional details.
+
+**Before you start**: Make sure to [get your API key](https://docs.getdbt.com/docs/dbt-cloud/dbt-cloud-api/user-tokens) and add it to the Authentication Header of your requests.
+
+Our example contains the following elements:
+
+![This diagram depicts how an API request works. The diagram depicts a Dev project and a Prod project. The Dev project contains two environments: a development and deployment environment. The deployment project contains a job. The diagram shows the deployment environment that contains the job being copied over to the Prod Project via API request.](/img/blog/2022-04-19-dbt-cloud-postman-collection/example-cloud-environment.png)
+
+### Setup
+
+Create a “development” and a “production” project in dbt Cloud using the UI
+Set up a [data warehouse connection](https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database) and [repository connection](https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories)
+
+#### Pull the environment from the development project
```

Comment:
The H4s here for 'environment' changes between capitalization of the 'E' in this one and the following one. (Same for the H4s involving Jobs.) Is this on purpose or do we want to use a consistent one across the H4s here?

Reply:
Good catch!